vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt

```diff
@@ -85,63 +55,49 @@ class UiAllScreensSanityTest {
     fun allScreensTest() {
         // Create an account
         val userId = "UiTest_" + UUID.randomUUID().toString()
-        uiTestBase.createAccount(userId = userId)
-
-        withIdlingResource(activityIdlingResource(HomeActivity::class.java)) {
-            assertDisplayed(R.id.roomListContainer)
-            closeSoftKeyboard()
+        elementRobot.login(userId)
+
+        elementRobot.settings {
+            general { crawl() }
+            notifications { crawl() }
+            preferences { crawl() }
+            voiceAndVideo()
+            ignoredUsers()
+            securityAndPrivacy { crawl() }
+            labs()
+            advancedSettings { crawl() }
+            helpAndAbout { crawl() }
         }
 
-        val activity = EspressoHelper.getCurrentActivity()!!
-        val uiSession = (activity as HomeActivity).activeSessionHolder.getActiveSession()
-
-        withIdlingResource(initialSyncIdlingResource(uiSession)) {
-            assertDisplayed(R.id.roomListContainer)
+        elementRobot.newDirectMessage {
+            verifyQrCodeButton()
+            verifyInviteFriendsButton()
         }
 
-        assertDisplayed(R.id.bottomNavigationView)
-
-        // Settings
-        navigateToSettings()
-
-        // Create DM
-        clickOn(R.id.bottom_action_people)
-        createDm()
-
-        // Create Room
-        // First navigate to the other tab
-        clickOn(R.id.bottom_action_rooms)
-        createRoom()
-
-        assertDisplayed(R.id.bottomNavigationView)
-
-        // Long click on the room
-        onView(withId(R.id.roomListView))
-                .perform(
-                        actionOnItem<RecyclerView.ViewHolder>(
-                                hasDescendant(withText(R.string.room_displayname_empty_room)),
-                                longClick()
-                        )
-                )
-        pressBack()
-
-        uiTestBase.signout()
-
-        // We have sent a message in a e2e room, accept to loose it
-        clickOn(R.id.exitAnywayButton)
-        // Dark pattern
-        clickDialogNegativeButton()
-
-        // Login again on the same account
-        waitUntilActivityVisible<LoginActivity> {
-            assertDisplayed(R.id.loginSplashLogo)
+        elementRobot.newRoom {
+            createNewRoom {
+                crawl()
+                createRoom {
+                    postMessage("Hello world!")
+                    crawl()
+                    openSettings { crawl() }
+                }
+            }
         }
 
-        uiTestBase.login(userId)
-        ignoreVerification()
+        elementRobot.roomList {
+            verifyCreatedRoom()
+        }
 
-        uiTestBase.signout()
-        clickDialogPositiveButton()
+// Disable until the "you don't have a session for id %d" sign out bug is fixed
+//        elementRobot.signout()
+////        Login again on the same account
+//        elementRobot.login(userId)
```

Comment:
We do not test regular login then... This is a bit annoying

Reply:
agreed! I'd like to look into the bug and reenable this ASAP